app/internal/page/certificateprovider/register.go

```diff
@@ -6,6 +6,8 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/notify"
+
```

Comment:
one day can you figure out why your import adding sometimes puts in blank lines

Reply:
Hahaha I'll ask JetBrains